lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -38,27 +52,42 @@ def on_block(node)
           describe, described_class, body = described_constant(node)
           return unless top_level_describe?(describe)
 
-          find_constant_usage(body, described_class) do |match|
-            add_offense(match, :expression, format(MSG, match.const_name))
+          @described_class = described_class
```

Comment:
Added a test case for this. It works with the multiple describes. Also added a comment